src/scenic/core/utils.py

```diff
@@ -143,6 +144,35 @@ def loadMesh(path, filetype, compressed, binary):
     return mesh
 
 
+def unifyMesh(mesh):
+    """Attempt to merge mesh bodies, aborting if something fails.
+
+    Should only be used with meshes that are volumes.
+    """
+    assert mesh.is_volume
+
+    # No need to unify a mesh with less than 2 bodies
+    if mesh.body_count < 2:
+        return mesh
+
+    mesh_bodies = mesh.split()
+
+    if not all(m.is_volume for m in mesh_bodies):
+        return mesh
+
+    try:
+        unified_mesh = trimesh.boolean.union(mesh_bodies, engine="scad")
```

Comment:
Actually, why are we using OpenSCAD here when the default for `MeshVolumeRegion.union` is Blender?